ocs_ci/ocs/node.py

```diff
@@ -1881,3 +1881,20 @@ def add_new_disk_for_vsphere(sc_name):
     ]
     node_with_min_pvs = min(num_of_pv_per_node_tuples, key=itemgetter(0))[1]
     add_disk_to_node(node_with_min_pvs)
+
+
+def get_odf_zone_count():
+    """
+    Get the number of Avaialbility zones used by ODF cluster
+
+    Returns:
+         int : the number of availability zones
```

Comment:
same: availability instead of avaialbility

Reply:
done.